imsvdex/vdex.py

```diff
@@ -119,8 +118,8 @@ def parse(self, file):
         """
         parses a VDEX vocabulary file or XML string
         """
-        if isinstance(file, StringTypes):
-            file = StringIO(file)
+        if isinstance(file, str):
+            file = BytesIO(file.encode("utf-8"))
```

Comment:
`file.encode("utf-8")` should be `safe_bytes(file)` otherwise we can double encode `str` in Python2.